resources/suitable/test_ns_dep.cljs

```diff
@@ -1,4 +1,7 @@
-(ns ^{:doc "Dependency of test-ns namespace"} suitable.test-ns-dep)
+(ns ^{:doc "Dependency of test-ns namespace"} suitable.test-ns-dep
+  (:require
+   ;; Exercises https://clojurescript.org/news/2021-04-06-release#_library_property_namespaces :
```

Comment:
I don't understand this comment, which makes me believe we can expand it a bit. :-)